src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3139,7 +3149,7 @@ public static AuditLogOptions getAuditLoggingOptions()
 
     public static void setAuditLoggingOptions(AuditLogOptions auditLoggingOptions)
     {
-        conf.audit_logging_options = auditLoggingOptions;
+        conf.audit_logging_options = new AuditLogOptions.Builder(auditLoggingOptions).build();
```

Comment:
nit: this line should be the only change in this file. Can all the import statement changes be reverted please.

Reply:
thanks, fixed locally, will push it sometimes